tests/unit/core/components/TestInstallButton.js

```diff
@@ -101,17 +108,18 @@ describe('<InstallButton />', () => {
       hasAddonManager: false,
     });
 
-    expect(root.type).toEqual('div');
-    expect(root.props.className).toContain(
-      'InstallButton InstallButton--use-button');
-    const buttonComponent = root.props.children[1];
-    expect(buttonComponent.type).toEqual(Button);
-
-    const props = buttonComponent.props;
-    expect(props.children).toEqual('Add to Firefox');
-    expect(props.className).toContain('InstallButton-button');
-    expect(props.className).not.toContain('Button--small');
-    expect(props.href).toEqual(installURL);
+    expect(root.type()).toEqual('div');
+    expect(root.hasClass('InstallButton')).toBeTruthy();
```

Comment:
This should probably use https://github.com/blainekasten/enzyme-matchers#tohaveclassnameclassnamestring since you have jest-enzyme.